ISSUES.md

```diff
@@ -10,6 +10,20 @@ Detailed information about each Bundler command, including help with common prob
 
 ## Troubleshooting
 
+### Permission denied when installing bundler
+Certain operating systems such as MacOS and Ubuntu have versions of Ruby that requires evelated privileges to install gems.
+
+```
+ERROR:  While executing gem ... (Gem::FilePermissionError)
+    You don't have write permissions for the /Library/Ruby/Gems/2.0.0 directory.
+```
+
+This is typically solved by prepending the command with `sudo`.
```

Comment:
awesome, i didn't know about this. Maybe we have both as suggestions here. I can see use cases where people need to install bundler as system and user gems.